pkgs/development/tools/gotools/default.nix

```
@@ -2,13 +2,12 @@
 
 buildGoModule rec {
   pname = "gotools-unstable";
```

Is it `unstable` if we're using a tag?
```suggestion
  pname = "gotools";
```

---

Good catch, thanks. Fixed in the same push that added the meta section.